docs/examples/feature_embedding.ipynb

```diff
@@ -0,0 +1,432 @@
+    "\n",
+    "# #change alternatives\n",
+    "# !sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.7 0\n",
+    "\n",
+    "# !pip install \"feathub-nightly[flink]\" sentence-transformers plotly matplotlib"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import Python dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from typing import Any, List\n",
+    "\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import requests\n",
+    "from sentence_transformers import SentenceTransformer\n",
+    "\n",
+    "from feathub.common import types\n",
+    "from feathub.feathub_client import FeathubClient\n",
+    "from feathub.feature_tables.sinks.memory_store_sink import MemoryStoreSink\n",
+    "from feathub.feature_tables.sources.file_system_source import FileSystemSource\n",
+    "from feathub.feature_tables.sources.memory_store_source import MemoryStoreSource\n",
+    "from feathub.feature_views.derived_feature_view import DerivedFeatureView\n",
+    "from feathub.feature_views.feature import Feature\n",
+    "from feathub.feature_views.on_demand_feature_view import OnDemandFeatureView\n",
+    "from feathub.feature_views.transforms.python_udf_transform import PythonUdfTransform\n",
+    "from feathub.table.schema import Schema"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download and preprocess resource files\n",
+    "\n",
+    "Download the hotel review dataset and append an incremental number column as\n",
+    "review IDs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "source_file_name = \"HotelReviews_data.csv\"\n",
+    "\n",
+    "if not os.path.exists(source_file_name):\n",
+    "    url = (\n",
+    "        \"https://raw.githubusercontent.com/Azure-Samples/azure-search-sample-data/main/hotelreviews/\"\n",
+    "        + source_file_name\n",
+    "    )\n",
+    "    r = requests.get(url)\n",
+    "    open(source_file_name, \"wb\").write(r.content)\n",
+    "    df = pd.read_csv(source_file_name)\n",
+    "    os.remove(source_file_name)\n",
+    "    df[\"reviews_id\"] = df.index\n",
+    "    df.to_csv(source_file_name, index=False, header=False)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Initialize FeatHub client"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client = FeathubClient(\n",
+    "    props={\n",
+    "        \"processor\": {\n",
+    "            \"type\": \"flink\",\n",
+    "            \"flink\": {\n",
+    "                \"deployment_mode\": \"cli\",\n",
```

Comment:
Should we use `session mode` with IP = local?

And can we combine `rest.address` and `rest.port` into one config named `master` similar to Spark?

Reply:
`rest.address` and `rest.port` is more friendly to users that have been familiar with Flink, so it might be better to keep them instead of introducing `master` configuration. I have add supports that when `rest.address` is `"local"`, FlinkProcessor would submit the job to a mini cluster.